### ISSUE TITLE
No space or unknown token in dataset dictionary, leading to training data corruption

### ISSUE THREAD

**Issue by steffencastle** [nhthang9x/Very-deep-cnn-pytorch#1]
A major issue here is that spaces and unknown tokens aren't encoded as in the paper. As a result, inputs have all spaces removed. This is probably the cause of the reduced accuracy compared to the paper.

**Comment by steffencastle** [comment]
The first letter of each sentence is also cut off.
